src/components/donate/HeroSection.tsx

```diff
@@ -8,7 +8,8 @@ const HeroSection = () => {
       <Flex
         mb={{ initial: "30%", sm: "10%" }}
         height={"550px"}
-        style={{ background: "#1D2F73" }}
+        // color also used in sponsor section, so maybe save it as background?
```

Comment:
Sure!  Might be worth looking into Radix's Theme component if we have common background / foreground colors.